src/GroupProject.pro.user.c3e3161

```diff
@@ -0,0 +1,562 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is this file fore?

Reply:
It's a user setting file. It doesn't cause any problem.